pilot/pkg/serviceregistry/kube/controller/ambient/authorization.go

```diff
@@ -432,7 +432,7 @@ const (
 	httpRuleFmt string = "ztunnel does not support HTTP rules (%s require HTTP parsing), in ambient mode you must use waypoint proxy to enforce HTTP rules. %s"
 
 	httpDenyRuleBoilerplate  string = "Deny rules with HTTP attributes will be enforced without their HTTP components. This is more restrictive than intended."
-	httpAllowRuleBoilerplate string = "Allow rules with HTTP attributes will be empty and never match. This is more restrictive than requested."
+	httpAllowRuleBoilerplate string = "Allow rules with HTTP attributes will be omitted and therefore never match. This is more restrictive than requested."
```

Comment:
```suggestion
	httpAllowRuleBoilerplate string = "Within an ALLOW policy, rules matching HTTP attributes are omitted. This will be more restrictive than requested."
```

It's not an "allow rule", it's an allow policy.
Is this correct?

Most of our examples show one rule, with multiple conditions.  I don't want people to get confused between that, and a policy with multiple _different_ rules.